ctlearn/default_config_files/singleCNN_param.yml

```diff
@@ -0,0 +1,493 @@
+# Example config file containing all possible options for CTLearn, whether
```

Comment:
It would be better to update the existing example config. You may want to add a `TRN with parameter` model to the default CTLearn models

Reply:
okay makes sense